setup.py

```diff
@@ -43,6 +43,8 @@
     'pytest-cov>=1.8.0',
     'pytest-pep8>=1.0.6',
     'pytest>=2.8.0',
+    'projectwithsharedtask'
```

Comment:
@bruno314 please remove the package from PyPI too (https://pypi.python.org/pypi/projectwithsharedtask/0.2.0.dev20150000)